VimeoUpload/Extensions/AFURLSessionManager+Extensions.swift

```diff
@@ -42,6 +42,6 @@ extension AFURLSessionManager
     @available(iOS 8.0, *)
     func taskForIdentifier(identifier: Int) -> NSURLSessionTask?
     {
-        return self.tasks.filter{ $0.taskIdentifier == identifier }.first as? NSURLSessionTask
+        return self.tasks.filter{ $0.taskIdentifier == identifier }.first as NSURLSessionTask!
```

Comment:
![screen shot 2016-04-06 at 7 14 41](https://cloud.githubusercontent.com/assets/126405/14305667/4233a0d0-fbc7-11e5-969a-71b6bb3cbf99.png)

I am not sure if you read above but i am in no way using swift on a daily basis - so i might just have done something stupid. But logic drived me to do what you see here :) Feel free to update it!